components/Chatbar/components/PromptFolders.tsx

```diff
@@ -5,9 +5,9 @@ import { FolderInterface } from '@/types/folder';
 import HomeContext from '@/pages/api/home/home.context';
 
 import Folder from '@/components/Folder';
-import { PromptComponent } from '@/components/Promptbar/components/Prompt';
```

Comment:
using absolute paths is usually better than relative if only for readability / enforcing a logical structure. Same folder is less of an issue, but generally we should not be using relative imports.